doc/audit/index.rst

```diff
@@ -134,3 +143,87 @@ you can set
    PI_AUDIT_SQL_TRUNCATE = True
 
 in ``pi.cfg``. This will truncate each entry to the defined column length.
+
+.. _logger_audit:
+
+Logger Audit
+------------
+
+The *Logger Audit* module can be used to write audit log information to
+the Python logging facility and thus write log messages to a plain file.
```

Comment:
maybe add that we can utilize the whole of the python logging functionality

Reply:
will do.

Reply:
done in 8f1fd030